### PR TITLE
add compose & mxtest

### PR DESCRIPTION
**Pull request by saces**
compose / mxtest tha runs a devel version from source (bun run --watch)

### CODE REVIEW

**Review by lukeelten — APPROVED** [review]

**Comment by saces** [comment]
please do not merge until registration is fixed, should be done tomorrow.

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,9 @@
 /node_modules
 /data
 /config
-
+.env
```
Comment (jaller94) [inline .gitignore:4]:
```suggestion
/.env
```

`/frontend/.env` ist bewusst im Repo.

File: mxtest/pcas.override.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+version: '3.8'
+
```
Comment (jaller94) [inline mxtest/pcas.override.yaml:2]:
```suggestion
version: '3.8'

```

Docker compose considers version numbers a deprecated property.
https://github.com/docker/compose/issues/11628#issuecomment-2000072567

File: compose.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+version: '3.8'
+
```
Comment (jaller94) [inline compose.yaml:2]:
```suggestion
```

File: compose.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+services:
```
Comment (jaller94) [inline compose.yaml:3]:
```suggestion
services:
```

Docker compose considers version numbers a deprecated property.
https://github.com/docker/compose/issues/11628#issuecomment-2000072567

File: README.md [hunk 5]
```diff
@@ -75,3 +75,41 @@ Stages:
...
+
+The compose files are intended to be used with mxtest, to create the required .env file run `./make-env.sh`.
+If the .env already exists, the reqiured items are appended.
+
+Create file `compose.override.yaml`
+
+```yaml
+version: '3.8'
+
+services:
```
Comment (jaller94) [inline README.md:91]:
```suggestion
services:
```

File: mxtest/pcas.override.yaml [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+services:
```
Comment (jaller94) [inline mxtest/pcas.override.yaml:3]:
```suggestion
services:
```

**Pull request merged**